jenkins/e2e-image/e2e-runner.sh

```diff
@@ -432,6 +432,12 @@ elif [[ ! "${JOB_NAME}" =~ -pull- ]]; then
   fi
 fi
 
+if [[ -n "${PRIORITY_PATH:-}" ]]; then
+  export PATH="${PRIORITY_PATH}:${PATH}"
+fi
+echo "kubectl on path is:"
+kubectl version --client
```

Comment:
I'm actually going to take it out. Something is too subtle here and it broke all of the *other* PR builders when I tried this, and I'd rather just get the other thing fixed.